clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1742,6 +1742,15 @@ as ``-mlong-calls`` and ``-mno-long-calls``.
   }];
 }
 
+def SYCLIntelNumSimdWorkItemsAttrDocs : Documentation {
```

Comment:
Same as in `Attr.td` - it seems better to group the attributes that are processed in the like manner